kubectl-plugin/pkg/cmd/version/version_test.go

```diff
@@ -5,15 +5,12 @@ import (
 	"context"
 	"fmt"
 	"testing"
-	"time"
 
 	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util"
+	. "github.com/ray-project/kuberay/kubectl-plugin/pkg/util/client/fake"
```

Comment:
```suggestion
	clientFake "github.com/ray-project/kuberay/kubectl-plugin/pkg/util/client/fake"
```

Don't import all symbols.